hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/listener/ClientListenerRegistration.java

```diff
@@ -45,4 +45,12 @@ public EventHandler getHandler() {
     Map<Connection, ClientConnectionRegistration> getConnectionRegistrations() {
         return registrations;
     }
+
+    @Override
+    public String toString() {
+        return "ClientListenerRegistration{"
+                + "codec=" + codec
```

Comment:
if no overriden `toString` method for codec and handler, will this output be enough for your troubleshooting? It will be just pointer numbers.

Reply:
We will see the class names. And class names are enough to understand which listener is being printed.